app/views/employees/_form.html.erb

```diff
@@ -11,11 +11,11 @@
   </div>
   <div class="field">
     <%= f.label :password %>
-    <%= f.password_field :password, class: 'form-control', :placeholder => 'Enter your password...' %>
+    <%= f.password_field :password, class: 'form-control', :placeholder => 'Enter new password...' %>
```

Comment:
Why this change? Will it not be confusing if the person tries to sign up for the first time? Or this is explicitly for changing information?